docs/kgctl.md

```diff
@@ -6,8 +6,10 @@ This tool can be used to understand a mesh's topology, get the WireGuard configu
 
 ## Installation
 
-Installing `kgctl` currently requires building the binary from source.
-*Note*: the [Go toolchain must be installed](https://golang.org/doc/install) in order to build the binary.
+The `kgctl` binary is automatically compiled for Linux, macOS, and Windows for every release of Kilo and can be downloaded from [the GitHub releases page](https://github.com/squat/kilo/releases/latest).
+
+### Building from Source
+Kilo is written in Golang and as a result the [Go toolchain must be installed](https://golang.org/doc/install) in order to build the `kgctl` binary.
```

Comment:
Do we want to mention that you need to `git clone` Kilo and `cd` into it. Or if you don't want to clone it specify a version when running `go install` eg.
```
go install github.com/squat/kilo/cmd/kgclt@latest
```
People not used to the go tool chain might be confused, like me initially.

Reply:
yes makes perfect sense